tests/Events/SpanTest.php

```diff
@@ -14,19 +15,23 @@ class SpanTest extends SchemaTestCase
     /** @var Timer|MockObject  */
     private $timer;
 
+    /** @var TimerFactory|MockObject  */
+    private $timerFactory;
+
     /** @var EventBean|MockObject  */
     private $parent;
 
     public function setUp(): void
     {
         parent::setUp();
 
-        /** @var EventBean|MockObject $parent */
+        $this->timer = $this->createMock(Timer::class);
+        $this->timerFactory = $this->createMock(TimerFactory::class);
+        $this->timerFactory->method('newtimer')->willReturn($this->timer);
```

Comment:
Yeah. It should be that way in the class. PHPUnit normalizes method names to lowercase so the mock works regardless. I sort of don't like that but I'm sure Sebastian had a reason. I'll double check the class.

Reply:
Looks like I noticed this and fixed it on a feature branch I created from this one. If we decide to merge this I'll have another PR for that later work and it will all get cleaned up.

Reply:
I merged the downstream changes into this branch.